assets/app/views/directives/environment.html

```diff
@@ -3,10 +3,11 @@
     <tbody>
       <tr ng-repeat="env in envVars">
         <td>{{env.name}}</td>
-        <td>
+        <!-- Avoid awkward wrapping of long, unbroken values like PATH. -->
+        <td style="word-break: break-all;">
```

Comment:
yep break-word is what's not supported in IE 11

On Tue, Mar 8, 2016 at 4:01 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/directives/environment.html
> <https://github.com/openshift/origin/pull/7868#discussion_r55426516>:
>
> > @@ -3,10 +3,11 @@
> >      <tbody>
> >        <tr ng-repeat="env in envVars">
> >          <td>{{env.name}}</td>
> > -        <td>
> > +        <!-- Avoid awkward wrapping of long, unbroken values like PATH. -->
> > +        <td style="word-break: break-all;">
>
> break-all seems to be supported:
>
> https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
>
> Maybe break-word was the problem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7868/files#r55426516>.
>